chrome/content/zotero/xpcom/schema.js

```diff
@@ -477,26 +479,37 @@ Zotero.Schema = new function(){
 			return deferred.promise;
 		})
 		.then(function (updated) {
-			if (runRemoteUpdateWhenComplete) {
-				var deferred = Q.defer();
-				if (updated) {
-					if (Zotero.Prefs.get('automaticScraperUpdates')) {
-						Zotero.proxyAuthComplete
-						.then(function () {
-							Zotero.Schema.updateFromRepository(2, function () deferred.resolve());
-						})
-					}
-				}
-				else {
-					Zotero.proxyAuthComplete
+			if (!runRemoteUpdateWhenComplete) return;
```

Comment:
We'll have some sort of release — probably not beta, but possibly "Alpha" or "Preview" — by the end of the month. So yeah, it's probably easier to just do this against that.